uniplate/src/zipper.rs

```diff
@@ -91,6 +91,14 @@ impl<T: Uniplate> Zipper<T> {
         self.path.len()
     }
 
+    /// Returns the index of the focus among its siblings from left to right.
+    ///
+    /// Returns `None` if the focus is the root node.
+    pub fn siblings_index(&self) -> Option<usize> {
```

Comment:
I think this is an implementation detail, so we should make it `pub(crate)` instead of `pub`.

If you need this method for `treemorph`, we can make it `pub` with `#[doc(hidden)]` instead.

```suggestion
    pub(crate) fn siblings_index(&self) -> Option<usize> {
```